ci/test_mac.sh

```diff
@@ -9,9 +9,12 @@ source $THIS_DIR/scripts/set_git_info.sh
 
 export WORKSPACE=$GITHUB_WORKSPACE
 export SOURCE_ROOT=$GITHUB_WORKSPACE
+export SHOULD_GENERATE_COVERAGE_REPORT=1
+
 python3 --version
 python3 -m venv venv
 source venv/bin/activate
 pip3 install -U pip
 pip3 install -U snowflake-connector-python
-$THIS_DIR/container/test_component.sh
+echo "cloud provider $CLOUD_PROVIDER"
```

Comment:
we don't need it - it is shown in the script when account name is printed

Reply:
ok